frontend/src/components/pages/CampOverview/CampersTable/CampersTable.tsx

```diff
@@ -219,7 +221,10 @@ const CampersTable = ({
                     maxWidth="32px"
                   >
                     <CampersTableKebabMenu
-                      editCamperFunc={editModalOnOpen}
+                      editCamperFunc={() => {
```

Comment:
super super nit: Can we call these functions more along the lines of `onClick` -> eg. `onClickEditCamper`? and also abstract this to another method so that the TSX section is easier to parse

Reply:
hmm not sure what you mean by "Abstract this to another method". You mean make a function which calls the editModalOpen function?

Reply:
if so, i think it's fine and we can just merge it in? Not sure what the benefit would be tbh

Reply:
Yeah something like `onClickEditCamper={openModalForSelectedCamper}`, but it's mostly just the naming that I think is worth it. Your call, giving the ✅